From cb2e43b73c3f1533f948cfe5d7693b06ce2170ee Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 7 Jan 2014 17:36:16 +0800
Subject: [PATCH 284/602] ENGR00293914 Config:update new config parameter

Update some new config parameter referring to other vendors' in AOSP.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/init.rc | 33 ++++++++++++++++++++-------------
 1 file changed, 20 insertions(+), 13 deletions(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index bcc4c61..53607d0 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -652,12 +652,12 @@ service mdnsd /system/bin/mdnsd
     disabled
     oneshot
 
-service dhcpcd_wlan0 /system/bin/dhcpcd -ABKL
+service dhcpcd_wlan0 /system/bin/dhcpcd -aABDKL
     class main
     disabled
     oneshot
 
-service dhcpcd_p2p /system/bin/dhcpcd -ABKL
+service dhcpcd_p2p /system/bin/dhcpcd -aABKL
     class main
     disabled
     oneshot
@@ -667,11 +667,28 @@ service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
     disabled
     oneshot
 
-service dhcpcd_bt-pan /system/bin/dhcpcd -BKLG
+service dhcpcd_bt-pan /system/bin/dhcpcd -ABKL
+    class main
+    disabled
+    oneshot
+
+service iprenew_wlan0 /system/bin/dhcpcd -n
+    class main
+    disabled
+    oneshot
+
+service iprenew_p2p /system/bin/dhcpcd -n
+    class main
+    disabled
+    oneshot
+
+service iprenew_eth0 /system/bin/dhcpcd -n
+    class main
     disabled
     oneshot
 
 service iprenew_bt-pan /system/bin/dhcpcd -n
+    class main
     disabled
     oneshot
 
@@ -708,16 +725,6 @@ service wifi_mac /system/bin/sh /system/etc/check_wifi_mac.sh
     class late_start
     oneshot
 
-service iprenew_wlan0 /system/bin/dhcpcd -n
-    class main
-    disabled
-    oneshot
-
-service iprenew_p2p /system/bin/dhcpcd -n
-    class main
-    disabled
-    oneshot
-
 # virtual sdcard daemon running as media_rw (1023)
 # default 4 threads have better performance to 2 threads.
 service sdcard /system/bin/sdcard -t4 /data/media /mnt/shell/emulated 1023 1023
-- 
1.8.0

